src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -292,7 +295,8 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
     {
         for (uint8_t i = 0; i < TableColorNames.size(); ++i)
         {
-            _CurrentColorTable.GetAt(i).Color(colorScheme.Table()[i]);
+            _CurrentNonBrightColorTable.GetAt(i).Color(colorScheme.Table()[i]);
+            _CurrentBrightColorTable.GetAt(i).Color(colorScheme.Table()[i]);
         }
```

Comment:
A note about the architecture. A color scheme object has a `Table` property (see terminal settings model for more details). The `Table` is a 16-sized array of colors. Each color in that table aligns with the name in `TableColorNames`.

Here, the old code was taking the color from the color scheme object's table, and throwing it into the `_CurrentColorTable` list. In the new code, we want to have the non-bright colors go in the non-bright table, and the bright colors to go in the bright table. Again, `8` is the magic number here. Anything less than that is non-bright, anything greater is bright. Codify that.